sky/shell/platform/android/org/domokit/sky/shell/FlutterSemanticsToAndroidAccessibilityBridge.java

```diff
@@ -117,6 +115,8 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         // the views should be ordered top-to-bottom, tie-breaking
         // left-to-right (right-to-left in rtl environments), height,
         // width, and finally by list order.
+        // (What about something like a two-column layout? How likely is it that the
+        // natural view order isn't the right relative order of the views?)
```

Comment:
OK. It sounds fine to do this for 99% of cases but it'd probably be a good idea for developers to have a way to override this when necessary. You may want to share the same mechanism for developers to override the order for keyboard focus traversal too.